docs/siem/detections/detections-ui-exceptions.asciidoc

```diff
@@ -0,0 +1,159 @@
+[[detections-ui-exceptions]]
+[role="xpack"]
+== Rule exceptions and value lists
+
+To prevent the creation of unwanted alerts, you can add exceptions to detection
+and Endpoint rules. Exceptions contain the source event conditions that
```

Comment:
I think all rules are detection rules. The "Elastic Endpoint" rule is just a prebuilt detection rule.